supabase/functions/settings-crud/index.ts

```diff
@@ -0,0 +1,186 @@
+import { serve } from "https://deno.land/std@0.168.0/http/server.ts";
+import { createClient } from 'https://esm.sh/@supabase/supabase-js@2';
+import { corsHeaders, handlePreflight } from '../_shared/cors.ts';
+
```

Comment:
do all this settings needs to be retrieved at the same time?

can this just be done using a postgres function to read the right settings directly from supabase.